settings.gradle

```diff
@@ -8,6 +8,7 @@ rootProject.name = dirName
 List projects = [
   'build-tools',
   'build-tools:reaper',
+  'checkstyle',
```

Comment:
I did not put this in buildsrc because it *looks* like the logger usage check to me. I can certainly move it if you'd feel more comfortable with it in the buildSrc but I might need tips on how to reference it in that case.

Reply:
It probably should move as well. It's not production code, but the current configuration is probably just historical.